case_studies/dc2_cataloging/utils/variational_dist.py

```diff
@@ -11,11 +11,9 @@ def __init__(self, survey_bands, tile_slen):
         super().__init__(survey_bands, tile_slen)
 
         self.factor_specs = {
-            "on_prob": UnconstrainedBernoulli(),
-            "loc": UnconstrainedTDBN(),
-            "galaxy_prob": UnconstrainedBernoulli(),
+            "n_sources": UnconstrainedBernoulli(),
+            "locs": UnconstrainedTDBN(),
+            "source_type": UnconstrainedBernoulli(),
+            "star_fluxes": UnconstrainedLogNormal(dim=len(survey_bands)),
+            "galaxy_fluxes": UnconstrainedLogNormal(dim=len(survey_bands)),
```

Comment:
@YicunDuanUMich I updated the syntax for your case study, but I haven't tested it.